doc/source/user/quickstart.rst

```diff
@@ -339,10 +339,10 @@ existing array rather than create a new one.
     >>> b
     array([[3.51182162, 3.9504637 , 3.14415961],
            [3.94864945, 3.31183145, 3.42332645]])
-    >>> a += b                  # b is not automatically converted to integer type
+    >>> a += b                  # b is not automatically converted to integer type # doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
         ...
-    numpy.core._exceptions.UFuncTypeError: Cannot cast from dtype('float64') to dtype('int64')  # IGNORE_EXCEPTION_DETAIL
+    numpy.core._exceptions.UFuncTypeError: Cannot cast from dtype('float64') to dtype('int64')
```

Comment:
Is this necessary? What error message is refguide expecting?

Reply:
The previous version returns
```pytb
    Traceback (most recent call last):
      File "/usr/lib/python3.7/doctest.py", line 1329, in __run
        compileflags, 1), test.globs)
      File "<doctest quickstart.rst[6]>", line 1, in <module>
        a += b                  # b is not automatically converted to integer type
    numpy.core._exceptions.UFuncTypeError: Cannot cast ufunc 'add' output from dtype('float64') to dtype('int64') with casting rule 'same_kind'
/opt/default-py3-env/lib/python3.7/site-packages/numpy/core/_asarray.py:136: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  return array(a, dtype, copy=False, order=order, subok=True)
```

Reply:
The doctest directive is not necessary